make-release.sh

```diff
@@ -302,13 +305,20 @@ verifyContainerExistsWithTimeout ${REGISTRY}/${ORGANIZATION}/che-machine-exec:${
 verifyContainerExistsWithTimeout ${REGISTRY}/${ORGANIZATION}/che-devfile-registry:${CHE_VERSION} 60
 # shellcheck disable=SC2086
 verifyContainerExistsWithTimeout ${REGISTRY}/${ORGANIZATION}/che-dashboard:${CHE_VERSION} 60
+# shellcheck disable=SC2086
+verifyBranchExistsWithTimeout "https://github.com/che-incubator/configbump.git" ${BRANCH} 60
+# shellcheck disable=SC2086
+verifyBranchExistsWithTimeout "https://github.com/eclipse/che-jwtproxy.git" ${BRANCH} 60
+# shellcheck disable=SC2086
+verifyBranchExistsWithTimeout "https://github.com/che-incubator/kubernetes-image-puller.git" ${BRANCH} 60
```

Comment:
you have a timeout of 60 here, and of 1 in the check-branches.sh script. Do we reallllly want to block the build for an hour if the branch isn't created? Seems like 5 mins would be sufficient?

Reply:
yeah probably should be small, but not too small, since it starts a separate job, that might get stuck in a queue